### ISSUE TITLE
New filter for browsing music collection

### ISSUE THREAD

**Issue by jymyj** [fatg3erman/RompR#148]
It would be convenient, in particular when you have plenty of musical genres and a large collection, to browse it by genre and then by albumartist (according to your mood). That filter would be similar to the existing 'Artists' filter but the first key would be 'Genre' and a second key would be 'Albumartist'. Then for each albumartist you would find his albums (in a particular genre) as this is the case with the 'Artists' filter.

**Example**:
```
Classical
    |___ Mozart
    |         |___ La finta giardiniera
    |         |___ Requiem
    |___ Brahms
               |___ Ein Deutsches Requiem
               |___ Symphony No 1
```
An additional nice feature would be to see the artist's picture next to his name in the list (as you see the albumart next to the album title). In particular, artists' pictures could be retrieved from the music directory in a similar way albumarts are retrieved. Indeed, often users store all the albums of a given artists inside a same directory (the artist's directory). So, an artist's picture could be retrieved from his directory according to some common file names and extensions.

**Comment by fatg3erman** [comment]
I agree that would be nice, but i's actually ridiculously difficult :) I'll leave this open but I can't guarantee it would ever happen.

Artist pictures might be doable, but only if those artist pictures are stored alongside local media, which makes it inappliccable for people using Mopidy with internet sources - there is no free online resource for artist images so I can't do it like I do for album art.
